packages/components/src/components/post-collapsible/post-collapsible.tsx

```diff
@@ -29,17 +29,19 @@ export class PostCollapsible {
   @Element() host: HTMLPostCollapsibleElement;
 
   /**
-   * If `true`, the element is initially collapsed otherwise it is displayed.
+   * If `true`, the element is collapsed otherwise it is displayed.
    */
-  @Prop() readonly collapsed?: boolean = false;
+  @Prop({ mutable: true }) collapsed?: boolean = false;
 
   @Watch('collapsed')
-  validateCollapsed(newValue = this.collapsed) {
+  collapsedChange() {
     checkEmptyOrType(
-      newValue,
+      this.collapsed,
       'boolean',
       'The `collapsed` property of the `post-collapsible` must be a boolean.',
     );
+
+    void this.toggle(!this.collapsed);
```

Comment:
You call `collapsedChange()` in `componentDidLoad()`, probably to run the `checkEmptyOrType` functionality.
But this will call the current line of code as well right from the beginning.
Is this intentionally? Or do I miss something?

Reply:
Yes it is intentional, if the collapsible has the prop `collpased="true"` we want it to be initially closed, to do so the `toggle` method has to be called.